src/lib/device-online-state.ts

```diff
@@ -21,8 +21,8 @@ import {
 
 const { root, api } = sbvrUtils;
 
-export const getPollInterval = async (uuid: string) => {
-	const getPollIntervalForDevice = api.resin.prepare<{ uuid: string }>({
+const getPollIntervalForDevice = _.once(() =>
```

Comment:
There's no need to do it lazily as it's pretty cheap/quick - you might as well do it eagerly to simplify the code

Reply:
@Page- is mentioned in the description, if I do this inline it will error with `api.resin is undefined`.
I didn't really had in mind to use lazy loading, I just had to, in order to avoid the `undefined` issue, which seems import/setup order related.
Is that what you meant with "eagerly"?

Reply:
Oh right, I guess this file is required fairly early on, fair enough